### ISSUE TITLE
error when running a project from a URI

### ISSUE THREAD

**Issue by devenxu1985** [mlflow/mlflow#1543]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in MLflow)**:
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
- **MLflow installed from (source or binary)**:source 
- **MLflow version (run ``mlflow --version``)**:1.0.0
- **Python version**: 3.6
- **npm version (if running the dev UI):

### Describe the problem
I was install conda with version 4.7.5 and it activate a venv with the command which is "conda activate" but not the "source activate".  
so I got a error like :
``` shell
2019/07/03 17:49:33 INFO mlflow.projects: === Created directory /var/folders/n0/xdqvfk9d3vq247p1gpp7zj3r0000gn/T/tmph0pg8wpt for downloading remote URIs passed to arguments of type 'path' ===
2019/07/03 17:49:33 INFO mlflow.projects: === Running command 'source activate mlflow-3eee9bd7a0713cf80a17bc0a4d659bc9c549efac && python train.py 0.4 0.1' in run with ID 'e12ac60a370d4037aa9836639cbfeb8b' ===
bash: activate: No such file or directory
2019/07/03 17:49:33 ERROR mlflow.cli: === Run (ID 'e12ac60a370d4037aa9836639cbfeb8b') failed ===
```
I think the problem is the code in the file as below 
mlflow/projects/__init__py
``` python
def _get_conda_command(conda_env_name):
    activate_path = _get_conda_bin_executable("activate")
    # in case os name is not 'nt', we are not running on windows. It introduces
    # bash command otherwise.
    if os.name != "nt":
        return ["source %s %s" % (activate_path, conda_env_name)]
    else:
        return ["conda %s %s" % (activate_path, conda_env_name)]
```
so I can only run a project with `--no-conda`

**Comment by nlaille** [comment]
will be fixed by : https://github.com/mlflow/mlflow/pull/1500

**Comment by mateiz** [comment]
Thanks for reporting this. We'll definitely get a fix into MLflow 1.1 in the next few days.

**Comment by smurching** [comment]
@devenxu1985 yep thanks for raising this - until the 1.1 release I'd recommend trying the workaround in https://github.com/mlflow/mlflow/issues/1507#issuecomment-506214268 (updating your `PATH` to include the `activate` and `conda` executables)
